migtests/scripts/run-validate-assessment-report.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ $# -gt 2 ]
+then
+	echo "Usage: $0 TEST_NAME [env.sh]"
+	exit 1
+fi
+
+set -x
+
+export YB_VOYAGER_SEND_DIAGNOSTICS=false
+export TEST_NAME=$1
+
+export REPO_ROOT="${PWD}"
+export SCRIPTS="${REPO_ROOT}/migtests/scripts"
+export TESTS_DIR="${REPO_ROOT}/migtests/tests"
+export TEST_DIR="${TESTS_DIR}/${TEST_NAME}"
+export EXPORT_DIR=${EXPORT_DIR:-"${TEST_DIR}/export-dir"}
+
+export PYTHONPATH="${REPO_ROOT}/migtests/lib"
+
+# Order of env.sh import matters.
+if [ $2 != "" ] #if env.sh is passed as an argument, source it
+then
+    if [ ! -f "${TEST_DIR}/$2" ]
+	then
+		echo "$2 file not found in the test directory"
+		exit 1
+	fi
+	source ${TEST_DIR}/$2
```

Comment:
what is $2 here will be? It will good to explain that in a comment at the top of this script, about the args.

Reply:
Just copied this part from our other test scripts. Sure will add the comment.